src/content/web3/index.md

```diff
@@ -36,19 +36,19 @@ The Web 2.0 period began in 2004 with the emergence of social media platforms. I
 
 ## Web 3.0: Read-Write-Own {#web3}
 
-The premise of 'Web 3.0' was coined by [Ethereum](/what-is-ethereum/) co-founder Gavin Wood shortly after Ethereum launched in 2014. Gavin put into words a solution for a problem that many early crypto adopters felt: the internet required too much trust. That is, most of the internet that people know and use today relies on trusting a handful of private companies to act in the public's best interests.
+The premise of 'Web 3.0' was coined by [Ethereum](/what-is-ethereum/) co-founder Gavin Wood shortly after Ethereum launched in 2014. Gavin put into words a solution for a problem that many early crypto adopters felt: the Web required too much trust. That is, most of the Web that people know and use today relies on trusting a handful of private companies to act in the public's best interests.
 
 ![Decentralized node architecture, representing Web3](./web3.png)
 
 ### What is Web3? {#what-is-web3}
 
-Web3 has become a catch-all term for the vision of a new, better internet. At its core, Web3 uses blockchains, cryptocurrencies, and NFTs to give power back to the users in the form of ownership. [A 2020 post on Twitter](https://twitter.com/himgajria/status/1266415636789334016) said it best: Web1 was read-only, Web2 is read-write, Web3 will be read-write-own.
+Web3 has become a catch-all term for the vision of a new, better Web. At its core, Web3 uses blockchains, cryptocurrencies, and NFTs to give power back to the users in the form of ownership. [A 2020 post on Twitter](https://twitter.com/himgajria/status/1266415636789334016) said it best: Web1 was read-only, Web2 is read-write, Web3 will be read-write-own.
 
 #### Core ideas of Web3 {#core-ideas}
 
 Although it's challenging to provide a rigid definition of what Web3 is, a few core principles guide its creation.
 
-- **Web3 is decentralized:** instead of large swathes of the internet controlled and owned by centralized entities, ownership gets distributed amongst its builders and users.
+- **Web3 is decentralized:** instead of large swathes of the Web controlled and owned by centralized entities, ownership gets distributed amongst its builders and users.
```

Comment:
```suggestion
- **Web3 is decentralized:** instead of large swathes of the internet controlled and owned by centralized entities, ownership gets distributed amongst its builders and users.
```

Reply:
I guess it's not technically incorrect. Fine.